linera-core/src/chain_worker/actor.rs

```diff
@@ -142,6 +142,13 @@ where
         callback: oneshot::Sender<Result<(ChainInfoResponse, NetworkActions), WorkerError>>,
     },
 
+    /// Get a blob if it belongs to the current locked block or pending proposal.
+    DownloadPendingBlob {
```

Comment:
Just a thought - if there will be more than one "type" of blob being downloaded (pending, locked, proposed, etc.) then maybe we can have a single message `DownloadBlob` with a field denoting type/source for the blob.

Reply:
I was thinking about this, but went with the separate request, because `DownloadBlob` is handled directly by the proxy whereas `DownloadPendingBlob` has to be forwarded to a worker.